googleapis_auth/lib/src/exceptions.dart

```diff
@@ -33,3 +27,12 @@ class UserConsentException implements Exception {
   @override
   String toString() => [message, if (details != null) details].join(' ');
 }
+
+class InvalidServerResponseException implements Exception {
```

Comment:
Could we document this:
```dart
/// Thrown when response from authentication service is invalid.
///
/// This could indicate invalid credentials.
```

Could this happen  as a result of invalid credentials?

Reply:
Renamed to `ServerRequestFailedException`

Reply:
Just saw this. Will do a follow-up!